### PR TITLE
Tools: fix duplicate label for NMEA GPS and Airspeed drivers  

### PR DESCRIPTION
**Pull request by shiv-tyagi**
This fixes duplicate labels used in NMEA GPS and Airspeed driver features. Also adds an autotest to test duplicate labels in build_options.py and raise an error if we find any.

### CODE REVIEW

File: Tools/autotest/test_build_options.py [hunk 1]
```diff
@@ -313,8 +313,18 @@ def check_deps_consistency(self):
...
             self.get_disable_defines(feature, options)
 
+    def check_duplicate_labels(self):
+        '''check that we do not have multiple features with same labels'''
+        options = self.get_build_options_from_ardupilot_tree()
+        seen_labels = {}
+        for feature in options:
+            if seen_labels.get(feature.label, None) is not None:
+                raise ValueError("Duplicate entries found for label '%s'" % feature.label)
+            seen_labels[feature.label] = True
```
Comment (peterbarker) [inline Tools/autotest/test_build_options.py:323]: This could be done a little nicer with a `set` and `feature.label in seen_labels` - but this is fine.

**Review by peterbarker — APPROVED** [review]
LGTM

**Comment by peterbarker** [comment]
Merged, thanks!
